### DISCUSSION TITLE
Puffco app on steam deck

### DISCUSSION THREAD

**Discussion by Dobby36** [SteamDeckHomebrew/decky-loader#591] [Ideas]
How about a plugin that lets you controll the puffco app for a puffco peak pro dab rig. It has an app for android ios and win so idk how much intailed it would be to create but it connects through bluetooth and the screen must stay active until force closed to work properly.

See reference to app in thought:  https://www.puffco.com/pages/app-experience

**Comment by TrainDoctor** [comment]
We do not allow plugin suggestions here. Thank you.
